src/engine/input/InputManager.cs

```diff
@@ -93,6 +94,11 @@ public static void OnFocusLost()
             attribute.Key.FocusLost();
     }
 
+    public static void SetRebindingActive(bool isRebinding)
```

Comment:
This should be made into a property with a documentation string on it.

Reply:
Done. When should one prefer a property?
Always when the value is exposed outside of the class or even in other cases?

Reply:
It's pretty complicated (as I'm not a C# expert), but I always prefer having a property instead of writing boilerplate get and set methods.

Reply:
I mainly write C++ for my job so C# is also pretty new to me and I'm not used to getter and setter functions. So seems like a good baseline. Thanks for the advice :)